### PR TITLE
[P4-670] Alert groupings

### PR DESCRIPTION
**Pull request by teneightfive**
This change groups the flags to avoid duplicates being shown in the summaries.

This is particularly necessary for the way alerts are being pulled in from NOMIS.

### Before
localhost_3000_moves_b8dec44a-fd67-4969-a4ff-046e862609c7 (1)

localhost_3000_move_42e141d1-14a2-4f6f-9303-c12fea85870c (1)

### After
localhost_3000_moves_b8dec44a-fd67-4969-a4ff-046e862609c7 (2)

localhost_3000_move_42e141d1-14a2-4f6f-9303-c12fea85870c

### CODE REVIEW

File: common/presenters/assessment-answer-to-tag.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 const { TAG_CATEGORY_WHITELIST } = require('../../config')
 
 module.exports = function assessmentAnswerToTag(hrefPrefix = '') {
-  return function map({ key, title, category }) {
+  return function map({ title, category }) {
```
Comment (feedmypixel) [inline common/presenters/assessment-answer-to-tag.js]: is `map` the best name for this? It made me look for the `lodash` map import. Maybe `getTagDetail` or `buildTagDetail`?

File: common/presenters/move-to-card-component.test.js [hunk 2]
```diff
@@ -94,19 +94,19 @@ describe('Presenters', function() {
             expect(transformedResponse.tags).to.deep.equal({
               items: [
                 {
-                  href: '/move/12345#concealed_items',
+                  href: '/move/12345#Concealed items',
```
Comment (feedmypixel) [inline common/presenters/move-to-card-component.test.js]: feels like we should be using fragments without spaces. Or is this just me?

File: app/move/views/_includes/assessment.njk [hunk 3]
```diff
@@ -1,42 +1,51 @@
...
+      {% set items = [] %}
+      {% for answer in answers %}
+        {% set answerHtml %}
+          {% if answer.nomis_alert_description %}
+            <h4 class="govuk-!-margin-top-0 govuk-!-margin-bottom-2">
+              {{ answer.nomis_alert_description }}
+            </h4>
+          {% endif %}
+
+          {{ answer.comments if answer.comments else '' }}
```
[the remark this answers is not in the document]
Comment (teneightfive) [reply app/move/views/_includes/assessment.njk]: Good point, looks redundant :)

File: common/presenters/move-to-card-component.test.js (hunk 2 above)
Comment (teneightfive) [reply common/presenters/move-to-card-component.test.js]: Have updated this so that it will always use kebab case for fragments.

**Pull request merged**